src/style.css

```diff
@@ -2,0 +1,49 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src:
+    url(fonts/Avenir-Book.ttf) format('truetype'),
+    url(fonts/Avenir-Heavy.ttf) format('truetype'),
+    url(fonts/Avenir.ttc) format('opentype');
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: solid 1px #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
```

Comment:
Avoid fixing container sizes as it can lead to overflow or accidental scroll bars. In this case, the width is set to 100%, which is usually fine, but be cautious with fixed heights or width in pixels that may not accommodate content or screen sizes.